src/test/kotlin/no/nav/tilleggsstonader/klage/fullmakt/FullmaktServiceTest.kt

```diff
@@ -0,0 +1,43 @@
+package no.nav.tilleggsstonader.klage.fullmakt
+
+import io.mockk.every
+import io.mockk.mockk
+import no.nav.tilleggsstonader.klage.testutil.FullmektigStubs
+import no.nav.tilleggsstonader.kontrakter.fullmakt.FullmektigDto
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.Test
+
+class FullmaktServiceTest {
+    private val mockFullmaktClient = mockk<FullmaktClient>()
+    private val serviceUnderTest = FullmaktService(mockFullmaktClient)
+
+    @Test
+    fun `hentFullmektige skal filtere bort inaktive fullmakter`() {
+        mockFullmektigeRespons(listOf(FullmektigStubs.ikkeGyldigEnda, FullmektigStubs.utgått))
+        val resultat = serviceUnderTest.hentFullmektige(dummyIdent)
+        assertEquals(emptyList<FullmektigDto>(), resultat)
+    }
+
+    @Test
+    fun `hentFullmektige tar bare med fullmakter med Tilleggsstønad-tema`() {
+        mockFullmektigeRespons(listOf(FullmektigStubs.ikkeRelevantTema, FullmektigStubs.medFlereTemaer))
+        val resultat = serviceUnderTest.hentFullmektige(dummyIdent)
+        assertThat(resultat.size).isEqualTo(1)
+        assertEquals(resultat.first(), FullmektigStubs.medFlereTemaer)
```

Comment:
For litt bedre lesbarhet kan det være fint å splitte tester med mellomrom mellom "given-when-then"-delene, dvs 
```suggestion
        mockFullmektigeRespons(listOf(FullmektigStubs.ikkeRelevantTema, FullmektigStubs.medFlereTemaer))
        
        val resultat = serviceUnderTest.hentFullmektige(dummyIdent)
        
        assertThat(resultat.size).isEqualTo(1)
        assertEquals(resultat.first(), FullmektigStubs.medFlereTemaer)
```

Reply:
Done!